BeeSwift/Settings/SettingsViewController.swift

```diff
@@ -87,6 +90,19 @@ class SettingsViewController: UIViewController {
     func showLogsTapped() {
         self.navigationController?.pushViewController(LogsViewController(), animated: true)
     }
+    
+    @objc private func fetchUser() {
+        Task { [weak self] in
```

Comment:
```suggestion
        Task { [weak self] @MainActor in
```

Reply:
I assume I should wait for you to either commit this or resolve this conversation thread before merging.